examples/life-expectancy/life-expectancy/03_create_table_life_expectancy_2010_2015.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE life_expectancy_2010_2015 (
+    State varchar(32),
```

Comment:
```suggestion
    State varchar,
```

I would remove the maximum length parameter from all varchar types. 

When I ran the data job locally it failed because some values were longer than 32 characters.

Reply:
Just done